src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker/mediapicker.controller.js

```diff
@@ -247,6 +248,31 @@ angular.module('umbraco').controller("Umbraco.PropertyEditors.MediaPickerControl
             return true;
         }
 
+        function removeAllEntries() {
+            $scope.mediaItems.length = 0;// AngularJS way to empty the array.
```

Comment:
Can we not just set `$scope.mediaItems = [];` and `$scope.ids = [];` never seen this in our code so far to set the array length to be 0 @nielslyngsoe ?